lib/post.js

```diff
@@ -40,16 +42,19 @@ function postXPI(manifest, options) {
           reject(e);
         });
         client.on("close", function(hadError) {
-          console.log(hadError ? "Posting XPI to %s failed" :
-                      "Posted XPI to %s", postURL);
+          if (hadError) {
+            logger.error("Posting XPI to " + postURL + " failed");
+          } else {
+            logger.info("Posted XPI to " + postURL);
+          }
```

Comment:
And I guess I should also call out that I slightly changed the logic here, where I converted this to an `if..else` instead of a ternery statement, because it let me use `logger.error()` and `logger.info()` a bit more appropriately [said the guy with zero context].